src/ucp/core/ucp_ep.h

```diff
@@ -129,6 +129,9 @@ typedef struct ucp_ep_config_key {
      */
     ucp_md_map_t           reachable_md_map;
 
+    /* upper low boundary for zcopy accross available lanes */
+    size_t                 rma_bw_min_zcopy;
```

Comment:
it should not be part of the key, since it can be deduced from lane configuration. it should be part of ucp_ep_config_t

Reply:
ok, shifted